src/Microsoft.VisualStudio.Threading/ThreadingEventSource.cs

```diff
@@ -143,6 +144,7 @@ public void WaitSynchronouslyStop()
     /// <param name="requestId">The request id.</param>
     /// <param name="mainThreadAffinitized">The execution need happen on the main thread.</param>
     [Event(PostExecutionStartEvent, Level = EventLevel.Verbose)]
+    [UnconditionalSuppressMessage("Trimming", "IL2026:Members annotated with 'RequiresUnreferencedCodeAttribute' require dynamic access otherwise can break functionality when trimming application code", Justification = "We're only serializing primitive types.")]
     public void PostExecutionStart(int requestId, bool mainThreadAffinitized)
     {
         this.WriteEvent(PostExecutionStartEvent, requestId, Boxed.Box(mainThreadAffinitized));
```

Comment:
What does `Boxed.Box` do? Is that still a primitive type (from the justification).

Reply:
Yes, it just boxes a bool such that `true` and `false` get the same box every time instead of allocating a new object each time.